extensions/funqy/funqy-knative-events/runtime/src/main/java/io/quarkus/funqy/runtime/bindings/knative/events/filters/CEAttributeLiteralEqualsFilter.java

```diff
@@ -0,0 +1,84 @@
+package io.quarkus.funqy.runtime.bindings.knative.events.filters;
+
+import java.util.function.Predicate;
+
+import io.quarkus.funqy.knative.events.CloudEvent;
+
+public class CEAttributeLiteralEqualsFilter implements Predicate<Object> {
```

Comment:
Could we merge these two filters?

Reply:
also could be this `Predicate<CloudEvent>` ?

Reply:
I think it could be merged indeed. About the type to be set to CloudEvent it then collides with the main type which is `Predicate<Object>` as it resides outside of the Knative events module. So I kept it really generic (Object). If needed I could make it more specific by introducing an interface that the cloud event would implement. Not sure it's worth it....

Reply:
The idea behind having filters on the `FunctionInvoker` was to allow filtering on any binding and not just knative events. But if you don't see this to be needed we can certainly move this as you suggested. One question though - should be use invoker name as key instead of the complete invoker instance?

Reply:
>The idea behind having filters on the FunctionInvoker was to allow filtering on any binding and not just knative events. 

I see. However right now we only need filter in `Knative events`, so I would rather not do any changes to `Common`. Maybe there will change in future. We can always change it once we need it.

>One question though - should be use invoker name as key instead of the complete invoker instance?

:+1:  That's probably good idea (although whole object with its default "addressof" `equals()`/`hashCode()` should work too).



what is your opinion @patriot1burke

Reply:
@matejvasek all changes applied, please have a look again, thanks!